src/main/java/org/codehaus/mojo/license/AbstractDownloadLicensesMojo.java

```diff
@@ -219,6 +219,8 @@
     @Parameter( defaultValue = "${project}", readonly = true )
     private MavenProject project;
 
+    @Parameter( property = "httpOnly", defaultValue = "false")
```

Comment:
I think it would be least surprise for users if property was named `license.httpOnly`.

Reply:
+1